tuna/miopen/db/miopen_tables.py

```diff
@@ -42,6 +41,8 @@
 from tuna.miopen.db.session import Session
 from tuna.miopen.utils.metadata import DIR_MAP
 from tuna.miopen.db.benchmark import Model, Framework
+from tuna.db.tuna_tables import JobMixin
+from tuna.db.tuna_tables import enum
```

Comment:
Is this supposed to be JobEnum?

Reply:
TypeError: FinStep: cannot extend enumeration 'JobEnum'. we cannot extend the JobEnum.
we need to define a base class that has methods but no member and then extend this base class. So, we cannot use JobEnum directly.

Reply:
changes rolled back